prisma/schema.prisma

```diff
@@ -96,4 +96,19 @@ model File {
     seniorId     String   @db.ObjectId
     senior       Senior   @relation(fields: [seniorId], references: [id])
     Tags         String[]
-}
\ No newline at end of file
+}
+
+model ChapterRequest {
+  id                   String    @id @default(auto()) @map("_id") @db.ObjectId
+  approved             Approval  @default(PENDING)
+  firstName            String
+  lastName             String
+  universityEmail      String    @unique
+  phoneNumber          String
+  university           String
```

Comment:
Should university be unique too? Is there a case when we might have 2 universities of the same name?